rules/TaskVariableHasSpace.py

```diff
@@ -0,0 +1,19 @@
+from ansiblelint import AnsibleLintRule
+
+import re
+
+class TaskVariableHasSpace(AnsibleLintRule):
+    id = 'E305'
+    shortdesc = 'Variables should be enclosed by spaces "{{ foo }}"'
+    description = ''
+    tags = ['task']
+
+    compiled = re.compile(ur'{{(\w*)}}')
```

Comment:
Can you explain why this should do? Should it enforce ```{{ foo }}```? In that case I don't think this matches ```{{ foo}}``` or ```{{foo }}``` which you probably also don't want.